### ISSUE TITLE
Vertical shake doesn't work

### ISSUE THREAD

**Issue by CookieHCl** [hoovercj/vscode-power-mode#71]
Settings:
```
"editor.renderWhitespace": "all",
"powermode.enabled": true,
"powermode.enableStatusBarComboCounter": false,
"powermode.enableStatusBarComboTimer": false,
```

VSCode version: 1.50.0

![ezgif-3-1ea0f498b628](https://user-images.githubusercontent.com/28638469/96018961-d6f94900-0e86-11eb-953f-b9a509750261.gif)

**Comment by Gytobp** [comment]
i have the same problem

**Comment by LDY681** [comment]
Same issue here. Help appreciated.

**Comment by LDY681** [comment]
#81 I used this method. works for me so far.

**Comment by hoovercj** [comment]
Thanks for reporting this! This should be fixed in the just-shipped v3.0 even without needing workarounds like "render whitespace".
